packages/openassistants/openassistants/contrib/sqlalchemy_query.py

```diff
@@ -12,13 +12,14 @@
     OpasAssistantMessage,
     OpasFunctionMessage,
     OpasMessage,
-    SuggestedPrompt,
+    merge_content,
 )
 from openassistants.data_models.function_input import BaseJSONSchema, FunctionCall
 from openassistants.data_models.function_output import (
     DataFrameOutput,
     FollowUpsOutput,
     FunctionOutput,
+    SuggestedPrompt,
```

Comment:
I think it's bad form that we have two versions of `SuggestedPrompt`, but this is the "right" one for a return value.  

Suggestion: We should eliminate one of these or at least rename one of them.

Reply:
I think https://github.com/definitive-io/openassistants/blob/348984ba1993dec1e90fa020fc3d2e3f674c6221/packages/openassistants/openassistants/data_models/chat_messages.py#L10 can just be removed